std/experimental/allocator/building_blocks/region.d

```diff
@@ -878,3 +890,385 @@ version(Posix) @system unittest
     assert((() nothrow @safe @nogc => alloc.owns(c))() == Ternary.yes);
     assert((() nothrow @safe @nogc => alloc.owns(null))() == Ternary.no);
 }
+
+/**
+The threadsafe version of the `Region` allocator.
+Allocations and deallocations are lock-free based using the `cas`
+functionality from `core.atomic`.
```

Comment:
Use the `REF` for links

Reply:
Can you please help me with this?
I'm not sure how to use that for "core.atomic"
Thanks